Makefile

```diff
@@ -61,6 +145,9 @@ NETGO_CHECK=@strings $@ | grep cgo_stub\\\.go >/dev/null || { \
 	echo "    sudo go install -tags netgo std"; \
 	false; \
 }
+# The flags we are passing to go build. -extldflags -static for making a static binary, 
+# -linkmode external for linking external C libraries into the binary, -X main.version for telling the
```

Comment:
I think `-linkmode external` is to force the use of the `ld` linker, external to the Go toolchain, and the reason we need to do that is otherwise it doesn't make a sufficiently static executable that it will run on its own in a container.

Although we don't actually do that now.